fsspec/implementations/tests/test_local.py

```diff
@@ -628,6 +628,13 @@ def test_strip_protocol_expanduser():
     assert not LocalFileSystem._strip_protocol("./").endswith("/")
 
 
+def test_strip_protocol_no_authority():
+    path = "file:\\foo\\bar" if WIN else "file:/foo/bar"
+    stripped = LocalFileSystem._strip_protocol(path)
+    assert "file:" not in stripped
+    assert stripped.endswith("/foo/bar")
```

Comment:
Why not `== "/foo/bar"`, is there any other possibility?

Reply:
Because on Windows the absolute path includes a drive letter so becomes `C:/foo/bar`

Reply:
See failed windows run https://github.com/SaravananSathyanandhaQC/filesystem_spec/actions/runs/4530930619/jobs/7980467557

Reply:
Got it, thank you.